.github/workflows/build.yml

```diff
@@ -52,22 +52,22 @@ jobs:
       run: ./bin/phpunit -d memory_limit=-1 --coverage-clover clover.xml
 
     - name: Upload coverage to Codecov
-      if: ${{ matrix.php-versions  == '7.1' }}
+      if: ${{ matrix.php-versions  == '8.4' }}
       uses: codecov/codecov-action@v1
       with:
         token: ${{ secrets.CODECOV_TOKEN }}
 
   phar:
-    runs-on: "ubuntu-20.04"
+    runs-on: ubuntu-latest
```

Comment:
would that mean when ubuntu gets upgraded we automatically get the latest version? If that the case an approach where we explicitly set the version is preferred imo